website/templates/search.mako

```diff
@@ -203,6 +203,7 @@
         <h4><span data-bind="if: parent_title">{{ parent_title }} /</span> <a data-bind="attr.href: url">{{title }}</a></h4>
         <!-- /ko -->
 
+        <strong><span data-bind="text: 'Date Created: ' + date_created['local'], tooltip: {title: date_created['utc']}" {{ date_created['local'] | fit:500 }}></span></strong>
```

Comment:
Sorry this whole thing is an old commit I need to push again. Hold off a for a while. Sorry again!